lib/ex_aws/sqs/request.ex

```diff
@@ -0,0 +1,28 @@
+defmodule ExAws.SQS.Request do
+  @moduledoc false
+
+  def request(client, queue_name, action, params \\ []) do
+    {_, http_method} = ExAws.SQS.Impl |> ExAws.Actions.get(action)
+
+    headers = [
+      {"content-type", "application/x-www-form-urlencoded"},
+    ]
+
+    ExAws.Request.request(http_method, client.config |> url(queue_name, params), [], headers, client)
+    |> parse(client.config)
+  end
+
+  def parse({:error, result}, _), do: {:error, result}
+  def parse({:ok, body}, config) do
+    case config[:json_codec].decode(body) do
```

Comment:
Sounds good. I'll hold off on further review till you ping me so that I don't badger you with stuff :)